### PR TITLE
Changes requested

### PR DESCRIPTION
**Pull request by taylorsavegnago**
QA Review

List:

https://airtable.com/appsEN1RgrUk3oyon/tbluJ0kJjMnu1rtsx/viw9dl4sT7CdFVqvO/recL8MG3iRWKSsWF4?blocks=hide
https://airtable.com/appsEN1RgrUk3oyon/tbluJ0kJjMnu1rtsx/viw9dl4sT7CdFVqvO/recsxTOZFLuVstwsJ?blocks=hide
https://airtable.com/appsEN1RgrUk3oyon/tbluJ0kJjMnu1rtsx/viw9dl4sT7CdFVqvO/rec3TGE4Kys6p2mhI?blocks=hide

I also fixed the color on the title of the testimonial

### CODE REVIEW

File: pages/tenants/digital-leasing/data/testimonials.js [hunk 1]
```diff
@@ -7,7 +7,13 @@ export default {
         skew: "left",
         testimonialBackground: "ui_100",
         testimonialColor: "blue_900",
-        title: []
+        title: [
+            {
+            spans: [],
+            text: "What Avail Renters are Saying",
+            type: "heading1",
```
Comment (quaardvark) [inline pages/tenants/digital-leasing/data/testimonials.js:14]: Should be H2 probably

File: pages/tenants/online-rent-payments/data/testimonials.js [hunk 2]
```diff
@@ -1,34 +1,40 @@
...
+    background: "blue_900",
+    color: "ui_100",
+    skew: "left",
+    testimonialBackground: "ui_100",
+    testimonialColor: "blue_900",
+    title: [
+      {
+        spans: [],
+        text: "What Avail Renters are Saying",
+        type: "heading1",
```
Comment (quaardvark) [inline pages/tenants/online-rent-payments/data/testimonials.js:14]: Should probably be H2

**Review by quaardvark — CHANGES_REQUESTED** [review]

File: pages/tenants/online-rent-payments/data/testimonials.js (hunk 2 above)
Comment (taylorsavegnago) [reply pages/tenants/online-rent-payments/data/testimonials.js]: Done

File: pages/tenants/digital-leasing/data/testimonials.js (hunk 1 above)
Comment (taylorsavegnago) [reply pages/tenants/digital-leasing/data/testimonials.js]: Done

**Comment by quaardvark** [comment]
@taylorsavegnago could you make sure the build succeeds (you can test locally with `yarn build`) and resolve the conflict?

**Comment by taylorsavegnago** [comment]
> @taylorsavegnago could you make sure the build succeeds (you can test locally with `yarn build`) and resolve the conflict?

Refactored the codes according to eslint

File: pages/tenants/online-rent-payments/data/how-it-works.js [hunk 3]
```diff
@@ -62,7 +60,7 @@ export default {
           height: 444,
           width: 480,
         },
-        alt: "Schedule rent payments",
+        alt: null,
```
Comment (quaardvark) [inline pages/tenants/online-rent-payments/data/how-it-works.js:63]: I don't see anywhere in the tickets that we need to remove alt texts

File: src/components/organisms/Testimonials/cards/index.js [hunk 4]
```diff
@@ -29,7 +30,7 @@ function TestimonialsCards({
         <Box ref={animationIntersectionView}>
           {title &&
             cloneElement(title, {
-              sx: { ...STYLING.headline, ...title?.props?.sx },
+              sx: { ...STYLING.headline, ...title?.props?.sx, color },
```
Comment (quaardvark) [inline src/components/organisms/Testimonials/cards/index.js:33]: I think color props in Testimonials relates to the color of the text on the card, not to the title color

**Review by quaardvark — CHANGES_REQUESTED** [review]

File: pages/tenants/digital-leasing/data/hero.js [hunk 5]
```diff
@@ -10,8 +10,8 @@ export default {
     ],
     description: [
       {
-        type: "paragraph",
-        text: "No longer hunt down a fax machine or scanner to send your landlord your lease. Instead, sign the lease online from the comfort of your own home.",
+        type: "heading4",
```
Comment (quaardvark) [inline pages/tenants/digital-leasing/data/hero.js:13]: This should be a paragraph

File: src/components/organisms/Testimonials/cards/index.js (hunk 4 above)
Comment (taylorsavegnago) [reply src/components/organisms/Testimonials/cards/index.js:33]: image
image

@quaardvark check that we don't have a text on this card. Do you want I to change the prop to color_title? In my opinion, it is not needed. 

image
In addition, we have some props that are not used in the card, right?

File: pages/tenants/digital-leasing/data/hero.js (hunk 5 above)
Comment (taylorsavegnago) [reply pages/tenants/digital-leasing/data/hero.js]: Changed

File: pages/tenants/online-rent-payments/data/how-it-works.js (hunk 3 above)
Comment (taylorsavegnago) [reply pages/tenants/online-rent-payments/data/how-it-works.js]: It happened when I fixed the conflicts, my mistake. Fixed.

File: src/components/organisms/Testimonials/cards/index.js (hunk 4 above)
Comment (quaardvark) [reply src/components/organisms/Testimonials/cards/index.js:33]: I think we can use an analyze color util to check if the background is dark

Comment (taylorsavegnago) [reply src/components/organisms/Testimonials/cards/index.js]: If the background is dark, the title will be white or else will be gray

**Review by quaardvark — APPROVED** [review]

**Pull request merged**